ixmp_workflow_utils/aggregation.py

```diff
@@ -0,0 +1,108 @@
+import logging
+
+import pandas as pd
+from pyam.utils import isstr, KNOWN_FUNCS
+
+log = logging.getLogger(__name__)
+
+
+def _get_method_func(method):
+    """Translate a string to a known method"""
+    if not isstr(method):
+        return method
+
+    if method in KNOWN_FUNCS:
+        return KNOWN_FUNCS[method]
+
+
+def wavg(df):
```

Comment:
This should be deprecated in favour of using the actual pyam function

Reply:
In this PR I just moved it out of instance workflow.

Reply:
moved out of this PR to #2